package/api/src/controllers/v1/auth/login/mod.rs

```diff
@@ -57,41 +56,9 @@ pub async fn handler(
         .map_err(|e| ApiError::JWTGenerationFail(e.to_string()))?;
 
     // If the member list has not been added, add the member list
-    let res: CommonQueryResponse<Member> = CommonQueryResponse::query(
-        &log,
-        "gsi1-index",
-        None,
-        Some(100),
-        vec![("gsi1", Member::get_gsi1(body.email.clone()))],
-    )
-    .await?;
-
-    if res.items.len() == 0 {
-        let member: Member = get_member_data(db.clone(), body).await;
-
-        cli.upsert(member.clone())
-            .await
-            .map_err(|e| ApiError::DynamoCreateException(e.to_string()))?;
-    } else {
-        let item = res.items.first().unwrap();
-
-        if item.deleted_at.is_some() {
-            let member: Member = (
-                CreateMemberRequest {
-                    email,
-                    name: None,
-                    group: None,
-                    role: None,
-                },
-                item.id.clone(),
-            )
-                .into();
-
-            cli.upsert(member.clone())
-                .await
-                .map_err(|e| ApiError::DynamoCreateException(e.to_string()))?;
-        }
-    }
+    let _ = create_member(db, body)
```

Comment:
이거는 근데 왜 구조체 안에 있는 메소드가 아니고 그냥 함수로 구현되어 있어요?
코드를 수정할때마다 조금씩 리팩토링을 하면서 수정하면 좋을듯 해요. 우리가 각잡고 리팩토링을 하기가 힘들거든요.

```suggestion
    let _ = ctrl.create_member(db, body)
```